provider/cpu_utils.py

```diff
@@ -168,3 +169,35 @@ def check_if_vm_vcpus_match_qemu(vm):
     enabled_count = vm.cpuinfo.smp + (len(enabled_vcpu_devices) * vcpus_count)
 
     return check_if_vm_vcpu_match(enabled_count, vm)
+
+
+def check_hotplug_ovmf_support(params, test, plug=True, unplug=False):
+    """
+    Check if ovmf & qemu version support vcpu hotplug or unplug
+    :param params: Dictionary with the test parameters
+    :param test: QEMU test object
+    :param plug: whether hotplug vcpu
+    :param unplug: whether hot-unplug vcpu
+    """
+    if not params.get("ovmf", ""):
+        return True
+
+    qemu_binary = utils_misc.get_qemu_binary(params)
+    qemu_version = utils_qemu.get_qemu_version(qemu_binary)[0]
+    query_package_ovmf = params["query_package_ovmf"]
+    if plug:
+        required_qemu = params["required_qemu"]
```

Comment:
QEMU version already checked.

Reply:
As @PaulYuuu  mentioned, `required_qemu` be handled during env preprocess.   It means `required_qemu_plug` as my understanding.   To avoid a mixup usage, how do you think about drop it in framework and handle it in tp-qemu ?